src/course-outline/data/api.js

```diff
@@ -459,3 +460,28 @@ export async function dismissNotification(url) {
   await getAuthenticatedHttpClient()
     .delete(url);
 }
+
+/**
+ * Downloads the file of the exported tags
+ * @param {string} courseId The ID of the content
+ * @returns void
+ */
+export async function getTagsExportFile(courseId, courseName) {
+  // Gets exported tags and builds the blob to download CSV file.
+  // This can be done with this code:
+  // `window.location.href = exportTags(contentId);`
+  // but it is done in this way to known when the operation ends to close the toast.
+  const response = await getAuthenticatedHttpClient().get(exportTags(courseId), {
+    responseType: 'blob',
+  });
+
+  const blob = new Blob([response.data], { type: 'text/csv' });
+  const url = window.URL.createObjectURL(blob);
+
+  const a = document.createElement('a');
+  a.href = url;
+  a.download = `${courseName}.csv`;
+  a.click();
+
+  window.URL.revokeObjectURL(url);
+}
```

Comment:
There's a weird side effect of this change, where after I select "Export Tags", the `#export` stays in the URL, and so the "Export Tags" menu item stays selected, so I can't re-select it and export again unless I navigate somewhere else first.

I don't think this is important -- people are unlikely to want to export tags multiple times -- it's just annoying, so not a blocker for this PR.

https://github.com/openedx/frontend-app-course-authoring/assets/7556571/f94bb228-1360-48ab-83ae-c4f44c3f6daa

Reply:
Fixed: https://github.com/openedx/frontend-app-course-authoring/pull/995/commits/6ccf70557dfae63401161298b5a4f5d082ebb74a